header/light_variant.html

```diff
@@ -0,0 +1,354 @@
+                    <a
+                      class="nav-link dropdown-toggle"
+                      href="#"
+                      role="button"
+                      data-bs-toggle="dropdown"
+                      aria-expanded="false"
+                    >
+                      Select an option
+                    </a>
+                    <ul class="dropdown-menu">
+                      <li>
+                        <a class="dropdown-item" href="/option1"
+                          >Dropdown option 1</a
+                        >
+                      </li>
+                      <li>
+                        <a class="dropdown-item" href="/option2"
+                          >Dropdown option 2</a
+                        >
+                      </li>
+                      <li>
+                        <a class="dropdown-item" href="/option3"
+                          >Dropdown option 3: Lorem ipsum dolor sit amet</a
+                        >
+                      </li>
+                      <li>
+                        <a class="dropdown-item" href="/option4"
+                          >Dropdown option 4: Lorem ipsum dolor sit amet</a
+                        >
+                      </li>
+                      <li>
+                        <a class="dropdown-item" href="/option5"
+                          >Dropdown option 5: Lorem ipsum dolor sit amet</a
+                        >
+                      </li>
+                    </ul>
+                  </li>
+                </ul>
+                <hr class="d-md-none vw-100 my-0" />
+                <ul class="navbar-nav d-md-none gap-0 w-100">
+                  <li class="nav-item">
+                    <a id="bookmarks_nav" class="nav-link" href="/bookmarks">
+                      Bookmarks
+                      <span
+                        class="badge badge-secondary bg-secondary"
+                        data-role="bookmark-counter"
+                        >0</span
+                      >
+                    </a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link" href="/search_history">History</a>
+                  </li>
+                  <li class="nav-item">
+                    <a
+                      href="#feedback"
+                      class="nav-link"
+                      data-bs-toggle="collapse"
+                      data-bs-target="#feedback"
+                      >Feedback</a
+                    >
+                  </li>
+                </ul>
+              </nav>
+            </div>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <div class="container">
+      Note: the <code>data-form-type="other"</code> attribute is a workaround
+      for a bug in Dashlane password manager. See
+      <a href="https://github.com/sul-dlss/vt-arclight/issues/495"
+        >https://github.com/sul-dlss/vt-arclight/issues/495</a
+      >
```

Comment:
It would be great to have a bit of an explanation here about how to implement the light-variant.